rules/lodash-fp.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  plugins: [
+    'lodash-fp',
+  ],
+
+  // For complete listing of rules and what they do, check out the docs.
+  // See: https://github.com/jfmengels/eslint-plugin-lodash-fp
+  rules: {
+    // Let people use both `flow` and `compose`.
+    'lodash-fp/consistent-compose': 0,
+
+    // Prevent simple errors.
+    'lodash-fp/no-argumentless-calls': 2,
+
+    // Chain is bad.™
+    // See: https://medium.com/making-internets/9bc1f80d51ba
+    'lodash-fp/no-chain': 2,
+
+    // Avoid unnecessary function wrapping.
+    'lodash-fp/no-extraneous-function-wrapping': 2,
+
+    // Avoid the performance overhead of `compose(f)`.
+    'lodash-fp/no-single-composition': 2,
+
+    // Prefer `_.compact` over `_.filter(_.identity)`.
+    'lodash-fp/prefer-compact': 2,
+
+    // Prefer `constant(x)` over `function () { return x; }`.
+    'lodash-fp/prefer-constant': [2, {arrowFunctions: false}],
+
+    // Prefer `_.flatMap` over consecutive `_.map` and `_.flatten`.
+    'lodash-fp/prefer-flat-map': 2,
+
+    // Prefer `get('a.b.c')` over `a && a.b && a.b.c`.
+    'lodash-fp/prefer-get': 2,
+
+    // Use `identity` instead of `function (x) { return x; }`.
+    'lodash-fp/prefer-identity': [2, {arrowFunctions: false}],
+
+    // Stop people from using legacy `lodash`.
+    'lodash-fp/use-fp': 2,
```

Comment:
So using them alongside each other is fine from a payload size perspective (as `lodash-fp` basically just runs `rearg` everywhere). But I agree with @jjt here; when you decide to adopt `fp` it becomes way too easy for people to skirt using it and then someone has to go _well this would be better if you used fp_. Maybe there's a legitimate reason for not using it (legacy code) and this rule can easily be disabled on a per-file or per-project basis.